### ISSUE TITLE
Abnormal sperm motility

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9827]
**HPO term**
HP:0012206

**Suggested revision of textual definition**
The parent term of "Abnormal sperm motility" should be "Abnormal sperm structure", which summarises "Abnormal sperm motility" and "Abnormal sperm morphology". 
A synonyme for this term is "Asthenozoospermia". This synonym is currently spelled wrongly "Asthenospermia" and is used as a synonym for the term "Reduced sperm motility (HP:0012207)". However, "Asthenozoospermia" labels completely immotile sperm as well as decrease motile sperm.

**Comment by pnrobinson** [comment]
[Abnormal sperm motility HP:0012206](https://hpo.jax.org/app/browse/term/HP:0012206)
is not a structural (morphological) abnormality, it is a functional (physiological) abnormality (even though the ultimate etiology may be structural, what we observe is a function or lack thereof).
In general, phenotypic features that are observed over the course of time are considered functional by HPO.
We may want to revise the current structure though.

**Comment by MagoWyr** [comment]
I agree

**Comment by pnrobinson** [comment]
Asthenospermia  == the word "astheno" is Greek for weak and I think it is typically used to refer to reduced but not absent motility?
I will add Asthenozoospermia as synonym.

e.g., https://www.sciencedirect.com/topics/medicine-and-dentistry/asthenospermia
Asthenospermia is defined as <40% sperm motility or less than 32% with progressive motility.

**Comment by MagoWyr** [comment]
Asthenozoospermia includes both absent and reduced motility. But the name Asthenospermia is wrong, it is Asthenozoospermia.
________________________________
